lib/core/pktmbuf/pktmbuf.c

```diff
@@ -121,21 +123,54 @@ pktmbuf_pool_create(char *addr, uint32_t bufcnt, uint32_t bufsz, uint32_t cache_
 
     /* Make sure the mbuf size is cache aligned */
     if (CNE_CACHE_LINE_ROUNDUP(bufsz) != bufsz)
+        CNE_ERR_RET("bufsz is not cache aligned\n");
+
+    if ((!metadata && metadata_bufsz) || (metadata && !metadata_bufsz))
+        CNE_ERR_RET("metadata and metadata_bufsz are invalid\n");
+
+    /* Size of metadata buffer should be a multiple of a cacheline */
+    if (metadata && ((metadata_bufsz % CNE_CACHE_LINE_SIZE) != 0))
```

Comment:
Can we be consistent and use (CNE_CACHE_LINE_ROUNDUP(metadata_bufsz) != metadata_bufsz)? Or change the above line 128 to use this.

Reply:
Convert to use CNE_CACHE _LINE_ROUNDUP